packages/compiler-core/__tests__/transforms/vIf.spec.ts

```diff
@@ -425,6 +425,20 @@ describe('compiler: v-if', () => {
       expect(generate(root).code).toMatchSnapshot()
     })
 
+    test('template v-if w/ v-once', () => {
+      const {
+        root,
+        node: { codegenNode },
+      } = parseWithIfTransform(
```

Comment:
The `parseWithIfTransform` does not contain `transformOnce` inside, which makes the resulting snap look unreasonable, and does not contain v-once related code.
Let's pass `transformOnce` to `parseWithIfTransform` to make the snap more sense.